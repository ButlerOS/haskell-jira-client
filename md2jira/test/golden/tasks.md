# epic
## story

- [ ] a task
with left over

- [ ] a task
  with right

> with quote?

- [ ] a task
> with comment

- [ ] a task
  > with comment
