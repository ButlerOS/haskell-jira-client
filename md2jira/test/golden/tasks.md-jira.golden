EpicTitle: epic
EpicBody:


StoryTitle: story
StoryBody:
* ☐ a task
with left over
* ☐ a task
with right

bq. with quote?
* ☐ a task

bq. with comment
* ☐ a task
bq. with comment

